backend_main/static/src/components/ImageUploader.js

```diff
@@ -6,7 +6,15 @@ import AvatarEditor from 'react-avatar-editor';
 // import classNames from "classnames";
 
 class ImageUploader extends Component {
-	state = { hasImage: false, image_file: null, scaled_image: null };
+	state = { hasImage: false, image_file: null, image_url: "", scaled_image: null };
+
+	componentDidMount() {
+		if (this.props.image_url !== "")
+			this.setState({	
+				image_url: "https://***REMOVED***.s3.amazonaws.com/" + this.props.image_url,
```

Comment:
what's this?

Reply:
oh it's supposed to be the link to the image stored in amazon AWS. the bucket name part of the URL got wiped when Evan did a cleaning up of the confidential info in our codebase. for now I think it is fine though, because i am not done with the image upload functionality, and it doesn't really break anything per se.